### PR TITLE
Feature/countdown timer

### PR DESCRIPTION
**Pull request by MishShevtsov**
- Created and implemented the Countdown timer to the event details. 
- Implemented styling and responsive to our timer. 
- Fixed some bugs which hiding map on the event details page
- Made some improvements to EventItem.js

### CODE REVIEW

File: client/src/shared/components/UI/CountDownTimer.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+      <path
+        fill='none'
+        stroke='#16a085'
+        strokeWidth='4'
+        d={describeArc(50, 50, 48, 0, radius)}
+      />
+    </svg>
+  );
+
+  // From StackOverflow: https://stackoverflow.com/questions/5736398/how-to-calculate-the-svg-path-for-an-arc-of-a-circle
```
Comment (yushevchuk) [inline client/src/shared/components/UI/CountDownTimer.js]: Remove comment, please.

File: client/src/shared/components/UI/CountDownTimer.js [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+      largeArcFlag,
+      0,
+      end.x,
+      end.y
+    ].join(' ');
+
+    return d;
+  }
+
+  // From StackOverflow: https://stackoverflow.com/questions/10756313/javascript-jquery-map-a-range-of-numbers-to-another-range-of-numbers
```
Comment (yushevchuk) [inline client/src/shared/components/UI/CountDownTimer.js]: Remove comment, please.

File: client/src/shared/components/UI/CountDownTimer.js [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+  function mapNumber(number, in_min, in_max, out_min, out_max) {
+    return (
+      ((number - in_min) * (out_max - out_min)) / (in_max - in_min) + out_min
+    );
+  }
+
+  const { days, hours, minutes, seconds } = timeLeft;
+
+  // Mapping the date values to radius values
```
Comment (yushevchuk) [inline client/src/shared/components/UI/CountDownTimer.js]: Remove comment, please.

File: client/src/shared/components/UI/CountDownTimer.js [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+  const daysRadius = mapNumber(days, 365, 0, 0, 360);
+  const hoursRadius = mapNumber(hours, 24, 0, 0, 360);
+  const minutesRadius = mapNumber(minutes, 60, 0, 0, 360);
+  const secondsRadius = mapNumber(seconds, 60, 0, 0, 360);
+
+  return (
+    <div>
+      {/* <h1>Countdown</h1> */}
```
Comment (yushevchuk) [inline client/src/shared/components/UI/CountDownTimer.js]: Remove comment, please.

Comment (MishShevtsov) [reply client/src/shared/components/UI/CountDownTimer.js]: Done!

File: client/src/shared/components/UI/CountDownTimer.js (hunk 3 above)
Comment (MishShevtsov) [reply client/src/shared/components/UI/CountDownTimer.js]: Done!

File: client/src/shared/components/UI/CountDownTimer.js (hunk 2 above)
Comment (MishShevtsov) [reply client/src/shared/components/UI/CountDownTimer.js]: Done!

File: client/src/shared/components/UI/CountDownTimer.js (hunk 1 above)
Comment (MishShevtsov) [reply client/src/shared/components/UI/CountDownTimer.js]: Done!

**Pull request merged**